exe-unit/components/gsb-http-proxy/src/message.rs

```diff
@@ -0,0 +1,26 @@
+use crate::error::HttpProxyStatusError;
+use crate::response::GsbHttpCallResponseEvent;
+use serde_derive::{Deserialize, Serialize};
+use std::collections::HashMap;
+use ya_service_bus::{RpcMessage, RpcStreamMessage};
+
+#[derive(Clone, Debug, Serialize, Deserialize, Eq, PartialEq)]
+#[serde(rename_all = "camelCase")]
+pub struct GsbHttpCallMessage {
+    pub method: String,
+    pub path: String,
+    pub body: Option<Vec<u8>>,
```

Comment:
In future we should serialize it as https://docs.rs/serde_bytes/latest/serde_bytes/
Serde handles `Vec<u8>` in very ugly way. In json it is an array with each element being separate number.